rendezvous/README.md

```diff
@@ -186,7 +186,55 @@ register under a namespace of interest (eg: the relay namespace). This
 can be mitigated by requiring a Proof of Work scheme for client
 registrations.
 
-This is TBD before finalizing the spec.
+We define "difficulty" as the number of zero bytes at the front of the
+hash assuming a big endian encoding.
+
+If a rendezvous node decides that Proof of Work is required to complete
+a registration, it will send a response with the status code `E_POW_REQUIRED`
+together with a `challenge` and the `target_difficulty`.
+
+A client is then expected to follow up with a `ProofOfWork` message, computing
+a hash by concatenating the following:
+
+- The UTF8 bytes of the string `libp2p-rendezvous-pow`
+- The contents of `challenge`
+- The `ns` field
+- The `signedPeerRecord` field
+- The `PeerId` of the rendezvous point we are registering with
```

Comment:
Given the addition of `challenge`, I am considering dropping this because PoW hashes are now unique to the rendezvous point already.